src/docs/docs/overview.md

```diff
@@ -32,14 +32,14 @@ various "feed" technologies (RSS, Atom, CDF) to allow blog posts from different
 platforms to be aggregated into a single page that was constantly updated with
 the latest posts by people within a particular community.
 
-[Written in Python by Jeff Waugh and Scott James Remnant](https://people.gnome.org/~jdub/bzr/planet/devel/trunk/),
+Written in Python by Jeff Waugh and Scott James Remnant,
```

Comment:
Why did you remove the link ?

Reply:
It directs us to `404 Not Found Page`, and the content of this link I assume that it's already in [this](https://en.wikipedia.org/wiki/Planet_(software))